lib/MySQL_HostGroups_Manager.cpp

```diff
@@ -3151,6 +3286,195 @@ SQLite3_result * MySQL_HostGroups_Manager::SQL3_Free_Connections() {
 	return result;
 }
 
+void MySQL_HostGroups_Manager::p_update_connection_pool() {
```

Comment:
This function should NOT access `MyHostGroups` without calling `wrlock()` first.
See `MySQL_HostGroups_Manager::SQL3_Connection_Pool()` for reference.

Reply:
Whoops... thanks for catch, fixed.